AppLovinAdapter/src/main/java/com/chartboost/mediation/applovinadapter/AppLovinAdapter.kt

```diff
@@ -695,8 +695,8 @@ class AppLovinAdapter : PartnerAdapter {
         context: Context,
         partnerAd: PartnerAd,
         partnerAdListener: PartnerAdListener?,
-    ): Result<PartnerAd> {
-        return suspendCancellableCoroutine { continuation ->
+    ): Result<PartnerAd> = withContext(IO) {
+        return@withContext suspendCancellableCoroutine { continuation ->
```

Comment:
This isn't at `setUp()` though.

Reply:
yeah @bwised this is AppLovinAdapter#showRewardedAd